src/draw/vg_lite/lv_vg_lite_utils.c

```diff
@@ -1028,13 +1028,18 @@ void lv_vg_lite_disable_scissor(void)
     LV_VG_LITE_CHECK_ERROR(vg_lite_disable_scissor());
 }
 
+int flush_cnt;
```

Comment:
I assume this is a debugging variable "left-over". 
If not needed, can be removed.
If needed, then please move to top of file, and change to unsigned int (if it really can't be negative).